frontend/packages/dev-console/src/components/import/upload-jar-validation-utils.ts

```diff
@@ -0,0 +1,45 @@
+import * as yup from 'yup';
+import { TFunction } from 'i18next';
+import {
+  nameValidationSchema,
+  projectNameValidationSchema,
+  applicationNameValidationSchema,
+  deploymentValidationSchema,
+  serverlessValidationSchema,
+  limitsValidationSchema,
+  routeValidationSchema,
+  imageValidationSchema,
+  buildValidationSchema,
+  resourcesValidationSchema,
+} from './validation-schema';
+import { healthChecksProbesValidationSchema } from '../health-checks/health-checks-probe-validation-utils';
+
+const fileNameRegex = /\.(jar|JAR)$/i;
+
+export const fileUploadValidationSchema = (t: TFunction) =>
+  yup.object().shape({
+    name: yup
+      .string()
+      .matches(fileNameRegex, {
+        message: t('devconsole~Must be a JAR file.'),
+      })
+      .max(253, t('devconsole~Cannot be longer than 253 characters.'))
```

Comment:
Do we limit the file name? macOS allows 255 characters

<img width="927" alt="Screenshot 2021-03-16 at 1 59 32 PM" src="https://user-images.githubusercontent.com/9278015/111278725-e52ed980-865f-11eb-9667-d25097a958a7.png">

should we limit the name field as well?

Reply:
for filename have used 253, as that's the largest I see in code used for other scenarios, and for name field is a common field used across and has 253 as well (i see validation kicks in here on focus out, we can handle this as separate ticket as this is the case in master as well)

![image](https://user-images.githubusercontent.com/5129024/111289513-3ee8d100-866b-11eb-8b21-61144d9006e1.png)